.github/workflows/build.yml

```diff
@@ -17,11 +17,11 @@ jobs:
     - uses: actions/checkout@v1
     - uses: actions/setup-go@v2-beta
       with:
-        go-version: '^1.13.8'
+        go-version: '^1.14.1'
     - uses: actions/cache@v1
       with:
         path: /home/runner/go/pkg/mod
         key: go-mod
     - uses: goreleaser/goreleaser-action@v1
       with:
-        args: release --snapshot
+        args: release --snapshot --skip-sign
```

Comment:
I'm not sure about the `--skip-sign` here. Can you explain?

Reply:
This is the workflow that gets executed during PRs or for any other non-master branch. In PRs, especially from Forks, we don't have the signing private key loaded (and actually don't want to expose it), so we skip signing. There won't be any releases anyway, as it's a snapshot build. But we have to provide the flag since Goreleaser tries to sign it it even though publishing is disabled with --snapshot.